testing/tests/DevExpress.ui.widgets.editors/htmlEditorParts/mentionModule.tests.js

```diff
@@ -442,17 +440,6 @@ QUnit.module('Mentions module', moduleConfig, () => {
             items.push(i);
         }
 
-        if(IS_EDGE_BROWSER) {
-            this.$element.css({
-                fontSize: '14px'
-            });
-        } else {
-            this.$element.css({
```

Comment:
Seems like we should not remove this code

Reply:
fixed